src/main/java/com/uber/cadence/client/WorkflowOptions.java

```diff
@@ -124,9 +124,10 @@ public Builder setWorkflowId(String workflowId) {
      * conditions Cadence allows two workflows with the same domain and workflow id run
      * simultaneously.
      * <li>
-     * <ul>
-     *   AllowDuplicateFailedOnly is a default value. It means that workflow can start if
-     *   previous run failed or was cancelled or terminated.
+     *
+     *     <ul>
```

Comment:
nit: indent?

Reply:
This is somehow changed automatically by the code formatter through gradlew build.